### DISCUSSION TITLE
Rainbow Screen

### DISCUSSION THREAD

**Discussion by tipeslowly** [thegetty/quire#44] [Show & Tell]
Greetings from Hong Kong. For the past couple years, we have been gradually getting up to a speed with Quire, using the platform to help promote current exhibitions within our museum at Hong Kong University. The most recent of these projects covers an [exhibition](https://digitalpub.umag.hku.hk/textiles/) of pictorial silk textiles from our permanent collection.

More recently, our colleagues within the Library and University Press have begun to cut their teeth on Quire. The first project built by the library can be found [here](http://shencai.lib.hku.hk/). 

Based on part of a chapter from a developing manuscript by HKU Prof. Yang Yuanzheng on Confucian music and rites, the Quire site complements an ongoing exhibition of Song, Yuan and Ming dynasty manuscripts selected from our special collections. "From the Rainbow Screen" details the amazing calligraphy and poetry of the concubine and amanuensis Shen Cai.

**Comment by geealbers** [comment]
Thanks so much for sharing this @tipeslowly! The silks in particular are really beautiful (those colors!) and of course using Quire for exhibition material like this is right in its wheel house. So glad to see it working well for you. 

We've not yet used Quire for exhibitions at Getty, but the question of how to make exhibition material more available digitally and also how to make it useful and accessible beyond the life of the physical exhibition has long been a question for us, even before this past year's pandemic shutdowns.

And really exciting to see Quire expanding across the University to the library. You're obviously a great Quire ambassador!
